jdbc/src/main/java/org/springframework/jdbc/core/RowMapper.java

```diff
@@ -0,0 +1,9 @@
+package org.springframework.jdbc.core;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+public interface RowMapper<T> {
```

Comment:
인터페이스에 메소드가 하나만 선언되어있으니 함수형 인터페이스로 활용될 여지가 많은 것 같아요
명시적으로 `@FunctionalInterface`를 선언해주는건 어떤가요?
```suggestion
@FunctionalInterface
public interface RowMapper<T> {
```

Reply:
함수형 인터페이스로 명시해두는 것 좋은 것 같습니다 바로 반영할게요 ~ 👍